doc/classes/EditorInspector.xml

```diff
@@ -14,6 +14,12 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="get_edited_object">
+			<return type="Object" />
+			<description>
+				Returns the currently selected object in the Inspector dock.
```

Comment:
```suggestion
				Returns the object currently selected in this inspector.
```

Remember that this is not a singleton, many inspectors exist (though only the docked one is accessible currently).